src/docker/composeappengine.h

```diff
@@ -2,6 +2,7 @@
 #define AKTUALIZR_LITE_COMPOSE_APP_ENGINE_H_
 
 #include <string>
+#include <unordered_map>
```

Comment:
not sure i see where you've added code to use an unordered_map?

Reply:
Some left over, I tried to be smart with the state management but then decided that it's not worth doing it at this stage.